src/pages/payment.js

```diff
@@ -65,21 +65,26 @@ class CheckoutPaymentPage extends Component {
   }
 
   componentDidMount () {
-    if (!this.props.cart.shipping_address) {
-      return Router.push('/checkout/shipping-address')
+    const { cart, checkout } = this.props
+    if (!cart.shipping_address) {
+      if (checkout.paymentMethod === 'default') {
```

Comment:
For consistency, should we consider using either
```
if (thirdPartyPaymentMethods.includes(checkout.paymentMethod)) {
```
or
```
if (checkout.paymentMethod === 'default') {
```
across all pages?

Reply:
This has been addressed in the next PR. Saw the comment after the changes in Part 2